components-compose/src/main/java/uk/gov/hmrc/components/compose/molecule/multiColumnRowView/MultiColumnRowItem.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.hmrc.components.compose.molecule.multiColumnRowView
+
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+
+data class MultiColumnRowItem(
+    val text: String = "",
+    val modifier: Modifier = Modifier,
+    val textStyle: TextStyle
```

Comment:
Usually we just want the text as standard body text. I think we can add that as a default here so we don't have to add it every time, just in cases where we want something different.
```suggestion
    val textStyle: TextStyle = HmrcTheme.typography.body
```